common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataConfig.scala

```diff
@@ -22,25 +22,72 @@ object PhotographersList {
 
 object MetadataConfig {
   val staffPhotographers = Map(
+    // Current
+    "Alicia Canter"  -> "The Guardian",
+    "Bill Code"      -> "The Guardian",
+    "David Sillitoe" -> "The Guardian",
+    "Graham Turner"  -> "The Guardian",
+    "James Mann"     -> "The Guardian",
+    "Jill Mead"      -> "The Guardian",
+    "Rachel Vere"    -> "The Guardian",
+    "Roger Tooth"    -> "The Guardian",
+    "Sean Smith"     -> "The Guardian",
+
+    // Past
+    "Catherine Shaw"        -> "The Guardian",
+    "Dan Chung"             -> "The Guardian",
+    "Denis Thorpe"          -> "The Guardian",
+    "Don McPhee"            -> "The Guardian",
+    "E Hamilton West"       -> "The Guardian",
+    "Emma Baddeley"         -> "The Guardian",
+    "Frank Baron"           -> "The Guardian",
+    "Frank Martin"          -> "The Guardian",
+    "Garry Weaser"          -> "The Guardian",
+    "Graham Finlayson"      -> "The Guardian",
+    "Harriet St Johnston"   -> "The Guardian",
+    "James Michelson"       -> "The Guardian",
+    "John Reardon"          -> "The Guardian",
+    "Lorna Roach"           -> "The Guardian",
+    "Marcus Mays"           -> "The Guardian",
+    "Martin Argles"         -> "The Guardian",
+    "Millie Burton"         -> "The Guardian",
+    "Peter Johns"           -> "The Guardian",
+    "Rachel Vere"           -> "The Guardian",
+    "Richard Blake"         -> "The Guardian",
+    "Robert Smithies"       -> "The Guardian",
+    "Sean Gibson"           -> "The Guardian",
+    "Ted West"              -> "The Guardian",
+    "Tom Stuttard"          -> "The Guardian",
+    "Tricia De Courcy Ling" -> "The Guardian",
+    "Walter Doughty"        -> "The Guardian",
+
+    "David Newell Smith"    -> "The Observer",
+    "Tony McGrath"          -> "The Observer"
+  )
+
+  val contractedPhotographers: Map[String, String] = Map(
+    "Antonio Zazueta"     -> "The Guardian",
+    "Christian Sinibaldi" -> "The Guardian",
     "Christopher Thomond" -> "The Guardian",
-    "Sean Smith"          -> "The Guardian",
     "David Levene"        -> "The Guardian",
-    "David Sillitoe"      -> "The Guardian",
-    "Eamonn Mccabe"       -> "The Guardian",
-    "Felicity Cloake"     -> "The Guardian",
```

Comment:
Just checked with Robert and she's a freelance writer for us (food). He's going to check the contracts.

Either way she doesn't seem to put herself as byline so we couldn't catch it with an automated filter. Instead, we may want to just do a batch override of her rights (where she's the byline through the picdar metadata overrides) and mark her as a contributor photographer. Someone would then have to do the same going forward when they import or use her pics.